safehttp/plugins/xsrf/xsrf.go

```diff
@@ -94,7 +94,7 @@ func addCookieID(w *safehttp.ResponseWriter) (*safehttp.Cookie, error) {
 // In case of state changing requests (all except GET, HEAD and OPTIONS), the
 // interceptor checks for the presence of the XSRF token in the request body
 // (expected to have been injected) and validates it.
-func (i *Interceptor) Before(w *safehttp.ResponseWriter, r *safehttp.IncomingRequest, cfg interface{}) safehttp.Result {
+func (it *Interceptor) Before(w *safehttp.ResponseWriter, r *safehttp.IncomingRequest, cfg interface{}) safehttp.Result {
```

Comment:
nit: Why `it` instead of `i`? Single-letter receiver names are the recommended style.

Reply:
I made the modification to be consistent across the repository, since this was the only place where we used `i` instead of `it`. Especially in the functions from the `mux.go` file, it seemed a bit more readable to refer to interceptors as `it` instead of `i`.